layouts/partials/scripts.html

```diff
@@ -1,3 +1,9 @@
+/* inject params to window.hextra for context in js */
+<script>
```

Comment:
if we use hugo template in the `flexsearch.js`, then no need to inject this global variable here